app/build.gradle

```diff
@@ -19,16 +19,14 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-kapt'
 
 android {
-    compileSdkVersion 26
-    buildToolsVersion '26.0.1'
```

Comment:
It's still better to explicitly declare the `buildToolsVersion`. The newest one corresponding to version 27 is 27.0.3.